lib/ckb/wallet.rb

```diff
@@ -53,7 +53,7 @@ def get_unspent_cells
         if skip_data_and_type
           cells.each do |cell|
             output = api.get_live_cell(cell.out_point).cell
-            results << cell if (output.data.nil? || output.data == "0x") && output.type.nil?
+            results << cell if output.type.nil?
```

Comment:
can not use cell with data. need to check if the corresponding outputs_data exist.

Reply:
updated